openai/components/oai_finetune_pipeline_component.yml

```diff
@@ -0,0 +1,114 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineComponent.schema.json
+name: microsoft_com_azureml_oaie_finetune_pipeline_v2
+version: 0.0.1
+type: pipeline
+display_name: PipelineComponent for OpenAI Finetuning workflow
+description: Pipeline component for OpenAI finetuning workflow
+
+inputs:
+  model:
+    type: string
+    optional: False
+    default: Ada
+    description: GPT-3 model engine
+    enum:
+      - Ada
+      - Babbage
+      - Curie
+      - DaVinci
+      - Text-Ada-001
```

Comment:
Can you please sync with Bhawana for model names, we don't need inference only models in this list.

Reply:
Done